tensorflow_model_optimization/python/core/quantization/keras/BUILD

```diff
@@ -243,8 +243,8 @@ py_library(
         ":quantize_layer",
         ":quantize_wrapper",
         # tensorflow dep1,
-        "//tensorflow_model_optimization/python/core/quantization/keras/default_8bit:default_8bit_quantize_layout_transform",
         "//tensorflow_model_optimization/python/core/quantization/keras/default_8bit:default_8bit_quantize_registry",
+        "//tensorflow_model_optimization/python/core/quantization/keras/default_8bit:default_8bit_quantize_scheme",
```

Comment:
@Xhark 

Just a note that until a change is made under core/api, the API is not considered publicly available (in the API docs). Not sure if that was an intentional choice. 

@nutsiepully

Reply:
Good point. @Xhark - Can you add QuantizationScheme, and Default8BitQuantizationScheme to the list of available classes?

Thanks.

Reply:
Sounds good, I'll try to add that. thanks.